src/bindings/python/src/pyopenvino/core/version.cpp

```diff
@@ -48,4 +48,14 @@ void regclass_Version(py::module m) {
             :return: OpenVINO's minor version.
             :rtype: int
         )");
+
+    cls.def_property_readonly(
```

Comment:
Python version was missing PATCH_NUMBER that was outputted in C++ version information